### PR TITLE
Add compatibility table to readme

### PR DESCRIPTION
**Pull request by tuelsch**
It is just a basic table with version numbers I was able to test with, except safari (this is just a guess). I also included a link to the ongoing discussion about IE support, this needs to be updated depending on the outcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,6 +327,13 @@ zoom.addEventListeners('show', event => {
 })
 ```
 
+## Compatibility
```
Comment (francoischalifour) [inline README.md]: It'd be awesome if you add a new line after this heading with new changes coming soon.

**Comment by jhildenbiddle** [comment]
#35 has been updated, bringing IE10+ support.

- A `CustomEvent` polyfill is not required.
- A [`<template>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) polyfill is required for [custom templates](https://github.com/francoischalifour/medium-zoom#using-a-custom-template)

I did not dive into `<template>` support, but I can confirm that the combination of #35  + [template-element-polyfill](https://github.com/neovov/template-element-polyfill) allowed the demo, dropbox-paper-template, and facebook-template pages in the repo to function properly on IE10/11.

File: README.md [hunk 2]
```diff
@@ -327,6 +327,13 @@ zoom.addEventListeners('show', event => {
...
+| IE | Edge | Chrome | Firefox | Safari |
+|----|------|--------|---------|--------|
+| No | 12   | 64     | 58      | 11     |
```
Comment (jhildenbiddle) [inline README.md]: A few suggestions:

- IE10+ will be supported once #35 has been merged.
- Edge 12 requires the same `<template>` polyfill as IE10+ for custom template support. If we intend to note this for IE, we should do the same for Edge 12.
- The evergreen browser versions should list the earliest compatible versions.

I did a quick test of the examples in various browsers to determine the earliest compatible version. Taking the polyfill notice into account, an updated table would look more like this:

| IE | Edge | Chrome | Firefox | Safari |
|----|------|--------|---------|--------|
| 10+<sup>1</sup> | 12+<sup>2</sup> | 36     | 34      | 9+     |

<sup>1</sup> IE10+ requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).
<sup>2</sup> Edge 12 requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).

**Comment by jhildenbiddle** [comment]
A few suggestions:

- IE10+ will be supported once #35 has been merged.
- Edge 12 requires the same `<template>` polyfill as IE10+ for custom template support. If we intend to note this for IE, we should do the same for Edge 12.
- The evergreen browser versions should list the earliest compatible versions.

I did a quick test of the examples in various browsers to determine the earliest compatible version. Taking the polyfill notice into account, an updated table would look more like this:

| IE | Edge | Chrome | Firefox | Safari |
|----|------|--------|---------|--------|
| 10+<sup>1</sup> | 12+<sup>2</sup> | 36     | 34      | 9+     |

<sup>1</sup> IE10+ requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).
<sup>2</sup> Edge 12 requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).

**Comment by francoischalifour** [comment]
@tuelsch I'd need access to your repo to update this PR. (otherwise, do you mind if I take this PR over?)

**Comment by francoischalifour** [comment]
You rock guys 🙌

Merging and releasing soon after!

**Pull request merged**